oak-segment-azure/src/main/java/org/apache/jackrabbit/oak/segment/azure/AzureArchiveManager.java

```diff
@@ -61,6 +62,12 @@ public class AzureArchiveManager implements SegmentArchiveManager {
 
     private static final String CLOSED_ARCHIVE_MARKER = "closed";
 
+    public static final String COPY_BATCH_SIZE_PROP = "segment.azure.batch.copy.size";
+
+    private static final int DEFAULT_COPY_BATCH = 1000;
+
+    private static final int COPY_BATCH = Integer.getInteger(COPY_BATCH_SIZE_PROP, DEFAULT_COPY_BATCH);
```

Comment:
I would suggest removing `final` here. It might be hard to manipulate its value in tests, since, as it is now, it will be initialized when the class is loaded and cannot be changed.

Reply:
Yes, sorry, I indeed meant ```static```.